docs/getting-started.mdx

```diff
@@ -105,15 +105,54 @@ They are documented in the [API Reference](pathname:///api/v7/classes/Sequelize.
 
 ### Testing the connection
 
-You can use the `.authenticate()` function to test if the connection is OK:
+You can use the `.authenticate()` function to test if the connection is OK, it runs `SELECT 1+1 AS result` query to test:
 
 ```js
+const sequelize = new Sequelize("db", "user", "pass");
```

Comment:
The documentation references the sequelize instance hundreds of times. There is no need to re-create one every time IMO. It'd be nice to figure out a way to add a link to the "Connecting to a database" on the variable though